Signal/src/ViewControllers/DebugUI/DebugUIMessages.m

```diff
@@ -232,6 +232,10 @@ - (nullable OWSTableSection *)sectionForThread:(nullable TSThread *)thread
                         actionBlock:^{
                             [DebugUIMessages createSystemMessagesInThread:thread];
                         }],
+        [OWSTableItem itemWithTitle:@"Create messages with variety of timestamps"
+                        actionBlock:^{
+                            [DebugUIMessages createTimestampMessagesInThread:thread];
```

Comment:
Added a debug UI to generate incoming & outgoing messages with a variety of timestamps.

Reply:
👍